source/features/latest-tag-button.tsx

```diff
@@ -125,7 +100,7 @@ async function init(): Promise<false | void> {
 	});
 
 	const link = (
-		<a className="btn btn-sm btn-outline ml-0 flex-self-center css-truncate rgh-latest-tag-button" href={String(url)}>
+		<a className="btn btn-outline ml-0 flex-self-center css-truncate rgh-latest-tag-button" href={String(url)}>
```

Comment:
👎 Duplicate of https://github.com/sindresorhus/refined-github/pull/3356

Reply:
```suggestion
		<a className="btn btn-sm btn-outline ml-0 flex-self-center css-truncate rgh-latest-tag-button" href={String(url)}>
```